posthog/models/plugin.py

```diff
@@ -177,6 +179,7 @@ def __str__(self) -> str:
     __repr__ = sane_repr("id", "name", "organization_id", "is_global")
 
 
+@reversion.register()
```

Comment:
- We should ignore `error` on `PluginConfig`, which is just the cached last error message, not actual state. 
- The `exclude` columns on `Plugin` somehow doesn't make sense to me. As in, why `from_json` and not `from_web` (so not all deprecated fields)? Why at all?

Reply:
> not all deprecated fields

Ah I missed that

```
    # DEPRECATED: these were used when syncing posthog.json with the db on app start
    from_json: models.BooleanField = models.BooleanField(default=False)
    from_web: models.BooleanField = models.BooleanField(default=False)
```

the comment covered both lines

---

> Why at all?

Felt wasteful to store deprecated fields in the revision